politeiad/sample-politeiad.conf

```diff
@@ -12,6 +12,9 @@
 ; Enable testnet
 ;testnet=true
 
+; dcrdatahost specifies the ip and port of the dcrdata host
+; dcrdatahost=127.0.0.1:443
```

Comment:
The sample conf example values should use the default values.  It's helpful if you can look at the sample conf and see the default values of all config settings.

Reply:
I put the testnet url since the same config has `testnet=true`.